Add an extra test to create a file required by the tests that follow it
--- src/CMakeLists.txt.orig	2019-07-17 18:28:03.915813844 +0100
+++ src/CMakeLists.txt	2019-07-17 18:28:25.393042910 +0100
@@ -206,6 +206,7 @@
 add_test (T_statpoly              ./neper --rcfile none -T -n 2 -statpoly "id,x,y,z,true,body,state,area,vol,size,diameq,radeq,sphericity,convexity,dihangleav,dihanglemin,dihanglemax,dihanglelist,vernb,edgenb,facenb,neighnb,verlist,edgelist,facelist,npolylist,facearealist,faceeqlist,vercoolist,scaleid(1)")
 # add_test (T_statpoint)
 # add_test (T_checktess)
+add_test (T_id5                   ./neper --rcfile none -T -n 5 -id 1)
 add_test (T_loadtess              ./neper --rcfile none -T -loadtess n5-id1.tess -statcell x -format tess,tesr)
 add_test (T_loadtesr              ./neper --rcfile none -T -loadtesr n5-id1.tesr -statcell x -format tesr)
 
